### ISSUE TITLE
Bug: include/hide suspicious doesn't seem to do anything?

### ISSUE THREAD

**Issue by Loki44** [tgxn/lemmy-explorer#181]
**Describe the bug**
When I click the "include/hide suspicious button on the communities page, the text that says "showing 30,009 communities" doesn't change.

Additionally the expected behaviour is a little ambiguous.  Are "suspicious" comms supposed to be shown when "include suspicious" is shown, or after the user has clicked "include suspicious" and "hide suspicious" is showing?

**To Reproduce**
1. Go to https://lemmyverse.net/communities?nsfw=null
2. Click on hide/include suspicious
3. See nothing happen

**Expected behavior**
the number changes.

**Comment by tgxn** [comment]
You're right! I wonder what happened with that, there's still some instances being detected as sus, but they don't hide/show correctly.

It does seem to work on the instances page, though only shows an additional ~9 instances...

It was originally meant to filter out instances that would appear "first" in the sorting methods like users, or posts, because those instances ""faked" that data and didn't actually have that much activity.

I think since now the "Smart Sort" is based on the Fediseer data as well as instance-level metrics, I could probably re-adjust this metric.


All of that said, I don't have a lot of time lately to be spending on this project, so there no promises of when I'll have a chance to fix it.
